VAMobile/src/screens/HealthScreen/Pharmacy/RefillScreens/RefillRequestSummary.tsx

```diff
@@ -134,6 +128,7 @@ function RefillRequestSummary({ navigation }: RefillRequestSummaryProps) {
   }
 
   const getRequestSummaryItem = () => {
+    console.debug(refillRequestSummaryItems)
```

Comment:
Looks like this was left behind from local testing?

Reply:
yep, removed